src/styles/blocks/stopwatch.scss

```diff
@@ -0,0 +1,45 @@
+.stop-watch {
+  position: relative;
+  @include sizeWatch($watch-size);
+  border-radius: 50%;
+  background-color: $back-color-watch;
+  transition: $sizing-clock;
+
+  &__center {
+    position: absolute;
+    top: 50%;
+    left: 50%;
```

Comment:
You created a placeholder for positioning, why you didn`t use it?

Reply:
This element is not repeated anywhere, so there is no need to put it there.